README.md

```diff
@@ -109,7 +109,7 @@ Example:
 
 #### sink.http
 
-Send the incomming requests to the specified host.
+Send the incomming requests to a evaluable expression
```

Comment:
I feel this can be left out, or added as a side note. New user won't know what an "evaluable expression" is.

Reply:
fixed